### PR TITLE
{math}[GCCcore/11.3.0] primecountpy v0.1.0, primecount v7.9

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

- [x] requires #19349

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/p/primecountpy/primecountpy-0.1.0-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+version = '0.1.0'
+
+homepage = 'https://pypi.org/project/primecountpy'
+description = "This is a Cython interface to the C++ library primecount."
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['78fe7cc32115f0669a45d7c90faaf39f7ce3939e39e2e7e5f14c17fe4bff0676']
+
```
Comment (verdurin) [inline easybuild/easyconfigs/p/primecountpy/primecountpy-0.1.0-GCCcore-11.3.0.eb:13]: @migueldiascosta please add the missing build deps for `GCCcore`.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
jsczen3c1.int.jsc-zen3.fz-juelich.de - Linux Rocky Linux 9.3, x86_64, AMD EPYC-Milan Processor (zen3), Python 3.9.18
See https://gist.github.com/SebastianAchilles/1d405b47d8457883b362e20efe223417 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**